unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AudioSource/Handler/ECSAudioSourceComponentHandler.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System;
+﻿
+using Cysharp.Threading.Tasks;
+using System;
 using DCL.Controllers;
 using DCL.ECS7.InternalComponents;
 using DCL.ECSRuntime;
 using DCL.Helpers;
 using DCL.Models;
 using DCL.SettingsCommon;
+using System.Collections.Generic;
```

Comment:
I think `using Cysharp.Threading.Tasks;` and `using System.Collections.Generic;` are not needed, maybe a left over?